scripts/get-version

```diff
@@ -19,25 +20,42 @@ fi
 
 # If we have an exact tag, just use it.
 if git describe --tags --exact-match "${COMMITISH}" >/dev/null 2>&1; then
-    echo -n "$(git describe --tags --exact-match "${COMMITISH}")"
-    if [ ! -z "${DIRTY_TAG}" ]; then
-        echo -n "+${DIRTY_TAG}"
-    fi
-
-    echo ""
-    exit 0
-fi
-
+    EXACT=1
+    TAG=$(git describe --tags --exact-match "${COMMITISH}")
 # Otherwise, increment the minor version version (if the package is 1.X or later) or the
 # patch version (if the package is pre 1.0), add the -alpha tag and some
 # commit metadata. If there's no existing tag, pretend a v0.0.0 was
 # there so we'll produce v0.0.1-dev builds.
-if git describe --tags --abbrev=0 "${COMMITISH}" > /dev/null 2>&1; then
+elif git describe --tags --abbrev=0 "${COMMITISH}" > /dev/null 2>&1; then
     TAG=$(git describe --tags --abbrev=0 "${COMMITISH}")
 else
     TAG="v0.0.0"
 fi
 
+# Check to see if the latest tag is for a submodule and transform to semver tolerant
+# e.g: submodule/submodule/.../v.X.X.X
+# sdk/v1.1.1 -> v1.1.1
+IFS='/'
+read -a tagsplit <<< "$TAG"
```

Comment:
Nit: I guess it's not strictly necessary, but throughout this script we always use braces. Should we just use them here for consistency?

```suggestion
read -a tagsplit <<< "${TAG}"
```